.env.example

```
@@ -306,6 +306,19 @@ ANTHROPIC_API_KEY=sk-ant-your-key-here
 #   OPENAI_BASE_URL=https://qwen35-122b.completions.near.ai/v1
 
 
```

dotenv-linter (4.0.0)

[warning] [ExtraBlankLine] Extra blank line detected

(ExtraBlankLine)